Makefile

```diff
@@ -86,3 +86,6 @@ ifndef HAS_GOLANGCI
 	curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | sh -s -- -b ./_output/bin ${GOLANGCI_VERSION}
 endif
 	./_output/bin/golangci-lint run
+
+lint-chart:
+	helm lint ./charts/descheduler
```

Comment:
Does the `helm lint` command exit non-zero if it finds a linting error?

Reply:
Also you may need to add some code to download the helm binary. See an example of lines 85 to 87.

Reply:
Yes `helm lint` will exit with a non-zero code if it finds an error.